app/src/main/res/layout/activity_chat.xml

```diff
@@ -9,13 +9,14 @@
     tools:context=".feature.chat.ui.ChatActivity">
 
     <FrameLayout
-        android:id="@+id/frameLayout"
+        android:id="@+id/hedvigToolbar"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/chat_top_bar_height"
-        android:background="?colorSurface"
+        android:layout_height="wrap_content"
+        android:background="@color/color_transparent_chat_85"
```

Comment:
This is actually potentially dangerous (and perhaps the cause of the issue) - using `ColorStateList`s as background colors is only valid on like API 28+ or something crazy high like that. What you do to remedy this is to use a 'placeholder' background that is just something like a square with white or black or whatever (we have some examples of this in the Key Gear-feature) and then use `backgroundTint` to apply it. Try that out before we go to using static colors in `colors_light`/`colors_dark` 🙂

Reply:
That worked 🎉